Custom scripts for manuscript 'A towering genome: co-adaptations to extreme stature inferred from giraffe genome and gene-edited mice'

# 00.basic_scripts
including three scripts:
01.convertMaf2List.pl was used to get reference-query nucleotide pairs from the maf file.
02.lst2gene.pl was used to generate the CDS sequence in query species accroding to the gff file of reference genome and the alignment list file generated by the first script.
03.cds2aa.pl was used to convert coding sequences to protein sequences.

# 01.CNEE_analysis
files unmberd from 01 t0 16 were used to predict HCE regions with the multi-sequence alignment maf files.
files unmberd from 17 t0 24 were used to get HCE regions with the at least four unique sites among rumiannt species and have a higer zscore (>2.5).
files unmberd from 25 t0 29 were used to annotate genes around HCE regions and CNEE regions with the gff file of goat genome.

# 02.PAML_analysis
including two scripts:
